web/src/main/java/org/springframework/security/web/savedrequest/CookieRequestCache.java

```diff
@@ -113,11 +113,11 @@ public void removeRequest(HttpServletRequest request, HttpServletResponse respon
 		response.addCookie(removeSavedRequestCookie);
 	}
 
-	private static String encodeCookie(String cookieValue) {
+	protected String encodeCookie(String cookieValue) {
 		return Base64.getEncoder().encodeToString(cookieValue.getBytes());
 	}
 
-	private static String decodeCookie(String encodedCookieValue) {
+	protected String decodeCookie(String encodedCookieValue) {
 		return new String(Base64.getDecoder().decode(encodedCookieValue.getBytes()));
 	}
 
```

Comment:
Please change the copyright header at the top of the file to read `2022` instead of `2020`.